### DISCUSSION TITLE
'couldn't install torch'

### DISCUSSION THREAD

**Discussion by AndroidArt12** [community/community#36960] [Other Feature Feedback, Questions, & Ideas]
Hi complete newbie at this and no coding knowledge - was following Youtube instructions on how to install local version of Stable Diffusion courtesy of Scott DeWeiler, in cmd section pressed go key and within a minute got an error about torch couldn't be installed  - attached screengrab of details.  Any help - specifically any help someone  with no coding knowledge can put into place would be greatly appreciated. 

![error github torch](https://user-images.githubusercontent.com/116442352/197341763-ab3d397d-acc0-4a58-a7e4-2ecc6f2effeb.png)

**Comment by ossanna16** [comment]
Hi @AndroidArt12,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Torch. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
